sites/funmooc/VERSION

```diff
@@ -0,0 +1 @@
+0.10.1
```

Comment:
Why not using the following files instead:

- `sites/${SITE}/src/backend/setup.cfg` and/or `sites/${SITE}/src/backend/__init__.py`
- `sites/${SITE}/src/frontend/package.json`

Reply:
good idea

Reply:
ok... so since I'm already checking for modification to sites/${SITE}/src/backend we are already tracking changes on the backend changes.

I will add a check in the CI that the tag, backend and the frontend version are the same.